### ISSUE TITLE
Making use of audio file metadata

### ISSUE THREAD

**Issue by yaxu** [musikinformatik/SuperDirt#122]
It could be nice to be able to make use of audio file metadata such as cue points. 

These can be embedded in wav files.. It seems this isn't generally accessible to SC but might be on the way:
  https://scsynth.org/t/reading-file-markers/241/6

We have talked about making use of off-line analysis, such as onset detection to index inside audio files, in the past.

**Comment by telephon** [comment]
If you export markers from audacity, you could already use those. It is just a little unclear how the parameters would look like.

**Comment by jkbd** [comment]
Replay Gain would also be interesting to have. One could use it to predefine `amp` settings for a loudness balanced sample set.

**Comment by yaxu** [comment]
In the case of audacity regions, I suppose it would really be tidal rather than supercollider that needs this info, as the main event scheduler is there.. As you might want to set a sound in time relative to its middle or end

**Comment by jkbd** [comment]
> you might want to set a sound in time relative to its middle

Yes! I always missed this feature with samplers, for example to align a cymbal swell crescendo to a beat.

**Comment by yaxu** [comment]
Here in section 3 Bel refers to points (rather than regions) in a sound object as pivots, after Duthen and Stroppa, 
https://hal.archives-ouvertes.fr/hal-00134179/document
